src/service/repeat_message.py

```diff
@@ -11,28 +11,26 @@
 from telegram.ext import CallbackContext, ContextTypes
 
 from constants.callback_data import CALLBACK_REPEAT_COMMAND
-from core.send_message import edit_message, reply_message
+from core.send_message import edit_message, send_message
 
 
 async def repeat_message_after_1_hour_callback(update: Update, context: ContextTypes.DEFAULT_TYPE) -> None:
-    """
-    Add a repeat message task to the queue.
-    """
+    """Add a repeat message task to the queue."""
     query = update.callback_query
     data = query.message
-    context.job_queue.run_once(callback=repeat_message_job, when=timedelta(hours=1), data=data)
-    await edit_message(update=update, new_text=data.text_markdown_v2_urled)
+    context.job_queue.run_once(
+        callback=repeat_message_job, when=timedelta(hours=1), data=data, chat_id=query.message.chat.id
+    )
+    await edit_message(update=update, new_text=data.text_markdown)
     await query.answer()  # close progress bar in chat
 
 
-async def repeat_message_job(update: Update, context: CallbackContext) -> None:
-    """
-    Repeat delayed message.
-    Instead this function should use 'send message' function from Slava
-    Kramorenko
-    """
+async def repeat_message_job(context: CallbackContext) -> None:
+    """Repeat delayed message."""
     data = context.job.data
-    await reply_message(update=update, text=data.text_markdown_v2_urled, reply_markup=data.reply_markup)
+    await send_message(
+        bot=context.bot, chat_id=context.job.chat_id, text=data.text_markdown, reply_markup=data.reply_markup
+    )
```

Comment:
А если попробовать так вызывать контекст же не теряется, читаемость не уменьшается, а вот кода становится меньше. Как по мне, так лучше и чище, особенно если говорить про такие передачи аргументов `update=update`
```suggestion
    await send_message(context.bot, context.job.chat_id, data.text_markdown, data.reply_markup)
```

Если так нельзя писать, то сори) скажите только почему, чтобы я не нервничал когда вижу такой код

Reply:
В данном случае можно не именовать аргументы.  
Я так писал, потому что мы рефакторим существующий код, и в нем аргументы именовали. Чтобы сохранять единообразие.

Reply:
Посмотрел наш [codestyle](https://www.notion.so/codestyle-e8de810f6a994c13ac97677dacf12f5a) - там нет опеределенных правил. Я сам, порой, сомневаюсь, как лучше сделать. То ли именовать переменные, то ли нет

Reply:
Как насчёт отказаться от именованных аттрибутов? Как по мне, кода станет меньше и он будет  красивее. 
Нужно только зафиксирвоать в кодстайле. 
Ну, и это не значит, что нужно переписать всё, просто когда в коде будет затрагиваться какая-то функция, то от туде будет удаляться ненужные аргументы.

Reply:
Я не против. @cianoid ?

Reply:
Конечно. Кодстайл попозже обновлю, менять можно

Reply:
Положил в кодстайл https://www.notion.so/codestyle-e8de810f6a994c13ac97677dacf12f5a#dc3727e607ad415c96ce9948efa0fb08